components/BottomNavigation/src/private/MDCBottomNavigationItemView.m

```diff
@@ -123,7 +124,9 @@ - (void)centerLayoutAnimated:(bool)animated {
   if (self.titleBelowIcon) {
     CGPoint iconImageViewCenter =
         CGPointMake(CGRectGetMidX(self.bounds),
-                    CGRectGetMidY(self.bounds) - CGRectGetHeight(self.bounds) * 0.1f);
+                    CGRectGetMidY(self.bounds) - CGRectGetHeight(self.bounds) / 2 +
```

Comment:
Doesn't CGRectGetMidY(self.bounds) - CGRectGetHeight(self.bounds) / 2 == 0?
If so, eliminate this line?

Reply:
Done. Yay, algebra. :-)